docs/extensions/get-started/anatomy.md

```diff
@@ -53,7 +53,8 @@ Lens uses this ID to uniquely identify your extension.
   "license": "MIT",
   "homepage": "https://github.com/lensapp/lens-extension-samples",
   "engines": {
-    "lens": "^4.0.0"
+    "node": "^14.18.12",
+    "lens": "^5.4.0"
```

Comment:
Probably this would be better for documentation (as some required minimum) and since it's anyway converted to  `^5.4.*` internally in compatibility check.

```suggestion
    "lens": "5.4"
```